js/script.js

```diff
@@ -11,299 +11,317 @@ window.getMegamapArgs = function() {
   var pairs = argsStr.replace(/^\?/, '').split('&');
   var args = {};
   unesc = unescape;
-  if ( typeof(decodeURIComponent) != 'undefined' )
-      unesc = decodeURIComponent;
-  for ( var i = 0; i < pairs.length; ++i ) {
-      pair = pairs[i].split('=');
-      if (pair[0]) {
-          if (pair[1])
-             pair[1] = unesc(pair[1].replace(/\+/g, ' '));
-          var key = unesc(pair[0].replace(/\+/g, ' '));
-          if( key == "layer" ) {
-              if( !args[key] ) {
-                  args[key] = [];
-              }
-              args[key].push(pair[1]);
-          } else {
-              args[key] = pair[1];
-          }
+  if (typeof(decodeURIComponent) != 'undefined')
+    unesc = decodeURIComponent;
+  for (var i = 0; i < pairs.length; ++i) {
+    pair = pairs[i].split('=');
+    if (pair[0]) {
+      if (pair[1])
+        pair[1] = unesc(pair[1].replace(/\+/g, ' '));
+      var key = unesc(pair[0].replace(/\+/g, ' '));
+      if (key == "layer") {
+        if (!args[key]) {
+          args[key] = [];
+        }
+        args[key].push(pair[1]);
+      } else {
+        args[key] = pair[1];
       }
+    }
   }
   return args;
 }
 
 var args = getMegamapArgs();
 
 if (args.css) {
-    var _x = document.createElement("link");
-    _x.rel = "stylesheet";
-    _x.type = "text/css";
-    _x.href = args.css;
-    document.body.appendChild(_x)
+  var _x = document.createElement("link");
+  _x.rel = "stylesheet";
+  _x.type = "text/css";
+  _x.href = args.css;
+  document.body.appendChild(_x)
 }
 
 var lat = parseFloat(args.lat) || 0,
-    lng = parseFloat(args.lng) || 0,
-    zoom = parseInt(args.zoom) || 2,
-    locate = (args.gl !== "n"),
-    searchZoom = parseInt(args.searchZoom) || 8,
-    layerControl = args.lc !== "n",
-    layerControlAlwaysShown = args.lc !== "h",
-    addMarkerControl = args.amc !== "n",
-    hidePastEvents = args.hpe !== "n",
-    dateAttribute = args.dateAttr || "date",
-    layersToShow = args.layer,
-    markerClusterRadius = parseInt(args.clusterRadius) || 80;
-var map = new L.Map('map', {"zoomControl": false, "maxZoom": 16}).setView([lat, lng], zoom);
-if( locate ) { map.locate({setView: true, maxZoom: 10}); }
+  lng = parseFloat(args.lng) || 0,

```

Comment:
Here is the beginning of variables we removed from the window. I would not keep things this way and would do something else with them in the near future.